war/src/main/js/api/securityConfig.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Provides a wrapper to interact with the security configuration
+ */
+
+var jenkins = require('../util/jenkins');
+var jquery = require('jquery-detached');
+var wh = require('window-handle');
+
+// gets the window containing a form, taking in to account top-level iframes
+var getWindow = function($form) {
+	var $ = jquery.getJQuery();
+	var wnd = wh.getWindow();
+	$(top.document).find('iframe').each(function() {
+		var windowFrame = this.contentWindow;
+		var $f = $(this).contents().find('form');
+		if($f.length > 0 && $form[0] === $f[0]) {
+			wnd = windowFrame;
+		}
+	});
+	return wnd;
+};
+
+var buildFormPost = function($form) {
+	var wnd = getWindow($form);
+	var form = $form[0];
+	if(wnd.buildFormTree(form)) {
+		return $form.serialize() +
+			'&core:apply=&Submit=Save&json=' + $form.find('input[name=json]').val();
+	}
+	return '';
+};
+
+var getFormCrumb = function($form) {
+	var wnd = getWindow($form);
+	return wnd.crumb;
+};
```

Comment:
Same for the above two. I know they're not exported, but I think it helps this module to not have other noise in it.